packages/workbox-core/_private/fetchWrapper.mjs

```diff
@@ -102,20 +102,41 @@ const wrappedFetch = async ({
   const pluginFilteredRequest = request.clone();
 
   try {
-    const fetchResponse = await fetch(request, fetchOptions);
+    let fetchResponse = await fetch(request, fetchOptions);
     if (process.env.NODE_ENV !== 'production') {
       logger.debug(`Network request for `+
       `'${getFriendlyURL(request.url)}' returned a response with ` +
       `status '${fetchResponse.status}'.`);
     }
+
+    for (const plugin of plugins) {
+      if (pluginEvents.FETCH_DID_SUCCEED in plugin) {
+        fetchResponse = await plugin[pluginEvents.FETCH_DID_SUCCEED]
+            .call(plugin, {
+              request: pluginFilteredRequest,
```

Comment:
Can you also add `event`, so it's consistent with the changes in https://github.com/GoogleChrome/workbox/pull/1640?